chessence-frontend/src/components/Navbar/navbar.js

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from "react"
+import "./navbar.css"
+
+export default function Navbar() {
+  useEffect (() => {
+    const imgElement = document.querySelector('img[alt="Chessence"]');
+      imgElement.width = 72;
```

Comment:
Why are you manually setting image width in javascript and not in css?

Reply:
I think it should be done by setting an id on the element (e.g. `<img src={"/logo.png"} alt="Chessence" id="navLogo" />`), then setting the width and height in the css (e.g. `#navLogo {width: 72px; height 72px;}`)

Reply:
> I think it should be done by setting an id on the element (e.g. `<img src={"/logo.png"} alt="Chessence" id="navLogo" />`), then setting the width and height in the css (e.g. `#navLogo {width: 72px; height 72px;}`)

Got it; thanks! Couldn't figure out the css code for this; turns out it came down to the "#" in the front lol. It's good for review now.

Reply:
Looks good!